admin/react/assets/src/dashboard-app/pages/welcome/SearchBar.js

```diff
@@ -0,0 +1,103 @@
+import { __ } from "@wordpress/i18n";
+import { useEffect, useRef, useState } from "react";
+import apiFetch from "@wordpress/api-fetch";
+import useDebounce from "../../../utils/useDebounce";
+
+const SearchBar = ({ setSearchResults, setLoading }) => {
+	const [query, setQuery] = useState("");
+	const debouncedQuery = useDebounce(query, 700);
+	const [searchIcon, setSearchIcon] = useState(true);
+	const ref = useRef(null);
+
+	const handleSearch = (e) => {
+		const q = e.target.value;
+		if ( '' === q ) {
+			setSearchIcon(true);
+		} else {
+			setSearchIcon(false);
+		}
+		setLoading(true);
+		setQuery(q);
+	};
+
+	useEffect(() => {
+		apiFetch({
+			path: `/bsf-custom-fonts/v1/admin/settings?q=${debouncedQuery}`,
+		}).then((results) => {
+			if (results) {
+				setLoading(false);
+				setSearchResults(results);
+			}
+		});
+	}, [debouncedQuery]);
+
+	const loadSearchIcon = () => {
+		if (searchIcon) {
+			return (
+				<svg
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38